src/main/java/io/supertokens/webserver/api/emailpassword/SignUpAPI.java

```diff
@@ -70,9 +70,6 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
             JsonObject result = new JsonObject();
             result.addProperty("status", "OK");
             JsonObject userJson = new JsonParser().parse(new Gson().toJson(user)).getAsJsonObject();
-            if (super.getVersionFromRequest(req).equals("2.4")) {
-                userJson.remove("timeJoined");
-            }
```

Comment:
why is this removed?

Reply:
We no longer support `2.4`, (2.7 is the lowest) right now.